libbeat/docs/shared-kibana-config.asciidoc

```diff
@@ -0,0 +1,213 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/shared-kibana-config.asciidoc[]
+//////////////////////////////////////////////////////////////////////////
+
+[[setup-kibana-endpoint]]
+== Set up the Kibana endpoint
+
+Starting with Beats 6.0.0, the Kibana dashboards are loaded into Kibana
+via the Kibana API. This requires a Kibana endpoint configuration.
+
+You configure the endpoint in the `setup.kibana` section of the
++{beatname_lc}.yml+ config file.
+
+Here is an example configuration:
+
+[source,yaml]
+----
+setup.kibana.host: "localhost:5601"
+----
+
+[float]
+=== Configuration options
+
+You can specify the following options in the `setup.kibana` section of the
++{beatname_lc}.yml+ config file:
+
+//REVIEWERS: I've decided to use full paths when referring to options rather than using indentation in examples. WDYT? Makes the examples a tad harder to read but will help immensely with users who don't know where to put stuff in the yaml hierarchy.
+
+[float]
+==== `setup.kibana.host`
+
+The Kibana host where the dashboards will be loaded. This setting is required.
+The scheme and port default to `http` and `5601` if you don't set them
+explicitly. If you use a path after the port number, you must specify
+the scheme and port. For example: `http://localhost:5601/path`.
```

Comment:
The Kibana configuration is the same as the Elasticsearch output configuration (https://www.elastic.co/guide/en/beats/filebeat/6.x/elasticsearch-output.html#hosts-option).
The `host` can be an `URL` or `IP:PORT`.  For example: http://192.15.3.2, 192:15.3.2:5601, http://192.15.3.2:6701/path. If no port is specified, then 5601 is used.

Note: When a node is defined as an IP:PORT, the scheme and path are taken from the `protocol` and `path` config options.